pkgs/by-name/th/thcrap-steam-proton-wrapper/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  bash,
+  subversion,
+  gnome,
+}:
+with lib; let
```

Comment:
Top-level `with` is discouraged, see [this draft PR](https://github.com/NixOS/nixpkgs/pull/294383) and [this guide](https://nix.dev/guides/best-practices#with-scopes) for some reasoning.